docs/migration/4_0.md

```diff
@@ -1,3 +1,7 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+- `CmsComponentsService` constructor has changed.
```

Comment:
Maybe more info. What was removed and was was added

Reply:
👍